Wise_Owl_Library/Extensions/BookExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Wise_Owl_Library.Data.Dto;
+using Wise_Owl_Library.Models;
+using Wise_Owl_Library.Models.Entities;
+
+namespace Wise_Owl_Library.Extensions
+{
+    public static class BookExtension
+    {
+        public static Book ToBook(this BookDto bookDto) => new()
+        {
+            Id = bookDto.Id,
+            Title = bookDto.Title,
+            Price = bookDto.Price,
+            Stock = bookDto.Stock,
+            Authors = bookDto.Authors.Select(a => new Author { Id = a.Id, Name = a.Name }).ToList()
+        };
+
+        public static Book EntityToBook(this BookEntity bookEntity) => new()
```

Comment:
Ten název ti tu nesedí protože tohle by mělo být pravděpodobně v jiné třídě. Samostatné extension metodě pro BookEntity

Reply:
Asi bych to rozdělil a udělal 2 mappery jeden pro dto layer a druhý pro db layer